packages/babel-generator/package.json

```diff
@@ -15,6 +15,7 @@
     "babel-runtime": "^6.9.0",
     "babel-types": "^6.14.0",
     "detect-indent": "^3.0.1",
+    "jsesc": "^1.3.0",
```

Comment:
Looks like this is Node >= 4 so I don't think we can land this at the moment. We all agree that we want to bump to >= 4 AFAIK, but we won't be able to do it until we bump our major version, which we haven't really had the opportunity to plan yet.